docs/profiling.md

```
@@ -11,15 +12,15 @@
        ebpf:
-         debug:
+         advanced:
            env:
-             PROFILE_PORT: "6060"
+             PPROF_ADDR: "127.0.0.1:6060"
     ```
 
-2. If you are running OpenShift/Kubernetes, port-forward the pod that you want to profile.
+2. If you are running Kubernetes, port-forward the pod that you want to profile.
 
     ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)